doc/adr/0008-TimeRangeData-widget.md

```diff
@@ -0,0 +1,53 @@
+# 8. Time Range Data Widget
+
+Date: 2023-1-10
+
+## Status
+
+New
+
+## Context
+
+There has been a request for the ability for users to be able to select part of a trace by copy/pasting timestamp data.  An attempt was made to implement this feature, but a consensus could not be reached on the UI/UX.  Design needs to be agreed upon before any further implementation.
+
+The proposed design is subject to change based on feedback.
+
+### Proposed UX Design
+
+#### UX Flow
+
+A "Time Range Data" widget will be added to the left panel.
+
+![current](0008/widget-sidebar.png)
+
+The widget will display: View Start, View End, Selection Start, Selection End, and an Edit button.
+
+![current](0008/time-range-data-widget-1.png)
```

Comment:
can show when the selection is modified? Maybe that will enable the submit button? Also submit on enter?